packages/lesswrong/styles/_posts.scss

```diff
@@ -1548,6 +1548,15 @@ svg.drag-handle {
   @include posts-link-styling();
 }
 
+.posts-page-content-body-html {
+  table {
```

Comment:
Hmm, it seems like you either want to wrap the block below in this table block, or remove this, but having an empty CSS block seems bad

Reply:
Oops, I should've checked the patch before commit. I'll force-push the new version.